.appveyor.yml

```diff
@@ -0,0 +1,74 @@
+environment:
+#  nodejs_version: "0.12.7"
+  matrix:
+    # First group is 32 bit python 3.4
+    - PYTHON: "C:\\Python34"
+      PYTHON_VERSION: "3.4.3"
+      PYTHON_ARCH: "32"
+
+platform: x86
+
+configuration: Release
+
+init:
+  - cmd: ECHO Processor architecture - %PROCESSOR_ARCHITECTURE%
+  - cmd: wmic OS get OSArchitecture
+
+  # As AppVeyor has multiple python install, verify which one uses by default
+  - cmd: ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%
+  - cmd: python --version
+  - cmd: python -c "import struct; print(struct.calcsize('P') * 8)"
```

Comment:
It is used to find out if you are running 32 or 64 bit versions of Python. It is partially for debugging, but it goes together with all the other system information, so I don't see any harm in living it there.